convert-date.py

```diff
@@ -81,7 +84,7 @@ def from_diff(in_series, out_series) -> "ErrorCount":
             column = in_series.name
             row = out_errors[0]
             value = in_series[row]
-            return ErrorCount(column, row, value, len(out_errors), 1)
+            return ErrorCount(column, int(row), str(value), len(out_errors), 1)
```

Comment:
Why these changes? Is it possible to revert them? Or did you write a unit test that addresses a bug that was there previously?

Reply:
The types of `row` and `value` were not the ones that are allowed for i18n message arguments and an exception was thrown by ICU. Before, the python string replacement was hiding this fact.

Reply:
I think we should be explicit. I had no idea these weren't int and str.

`row` is a `numpy.int64`. Let's do this:

```
row = int(out_errors[0])  # np.int64 => int
```

`value` should always be a `str` because it comes from the not-null values of the input. The possibilities are:

* `in_series` is datetime => there is no way we get errors
* `in_series` is text => then this is str, right?
* `in_series` is numbers => then we shouldn't have gotten this far, because this module doesn't handle numbers.

To prevent `in_series` from ever being numbers, please add `column_types: [ datetime, text ]` to the multicolumn.

After that, we shouldn't need the `int()` or `str()` in the ErrorCount ctor call.

I'd be very grateful if you could test and make sure I'm right about this :).

Reply:
Indeed, you are right :)